Object.php

```diff
@@ -851,6 +851,9 @@ public function ingestDatastream(&$ds) {
       // Assume NewFedoraDatastreams will have a log message set.
       'logMessage' => ($ds instanceof NewFedoraDatastream) ? $ds->logMessage : "Copied datastream from {$ds->parent->id}.",
     );
+    if ($ds->checksum !== 'none') {
```

Comment:
I think an [`isset()`](https://github.com/Islandora/tuque/pull/158/files#diff-d9ededc5334628e2c15e2f1934472aa9R678) check would suffice here, without having to worry about the value when "unset".